apps/web/src/components/RecruitmentAvailabilityDialog/RecruitmentAvailabilityDialog.tsx

```diff
@@ -0,0 +1,192 @@
+import React from "react";
+import { FormProvider, useForm } from "react-hook-form";
+import { useIntl } from "react-intl";
+
+import { Dialog, Button } from "@gc-digital-talent/ui";
+import { RadioGroup } from "@gc-digital-talent/forms";
+import { toast } from "@gc-digital-talent/toast";
+
+import {
+  PoolCandidate,
+  useChangeApplicationSuspendedAtMutation,
+} from "~/api/generated";
+import { fullPoolTitle } from "~/utils/poolUtils";
+
+interface RecruitmentAvailabilityDialogProps {
+  candidate: PoolCandidate;
+}
+
+type FormValues = {
+  isSuspended: "true" | "false"; // Note: RadioGroup only accepts strings
+};
+
+const RecruitmentAvailabilityDialog = ({
+  candidate,
+}: RecruitmentAvailabilityDialogProps) => {
+  const intl = useIntl();
+  const [, executeMutation] = useChangeApplicationSuspendedAtMutation();
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const isSuspended = !!candidate.suspendedAt;
+  const title = fullPoolTitle(intl, candidate.pool);
+
+  const methods = useForm<FormValues>({
+    defaultValues: { isSuspended: isSuspended ? "false" : "true" },
+  });
+  const {
+    handleSubmit,
+    formState: { isSubmitting },
+  } = methods;
+
+  const updateSuspendedAtStatus = async (values: FormValues) => {
+    await executeMutation({
+      id: candidate.id,
+      isSuspended: values.isSuspended === "true",
```

Comment:
Hang on, maybe the above will pose a problem after all. 
As "I am available" maps to "true", this passes in `isSuspended = true`, so the opposite of how it should play out.
Selecting "I am available" will suspend your pool candidate.

Reply:
Yep, imported the component in to try it out

https://recordit.co/EJ1GVuk9IK